README.md

```diff
@@ -144,4 +144,4 @@ There is also a Dockerfile which helps for isolating build problems and local de
 2. Clone this repo,
 3. Run `./scripts/start_container.sh`
 
- This should start a local docker container called arviz, as well as a Jupyter notebook server running on port 8888. The notebook should be opened in your browser automatically (you can disable this by passing --no-browser). The container will be running the code from your local copy of arviz, so you can test your changes.
+This should start a local docker container called arviz, as well as a Jupyter notebook server running on port 8888. The notebook should be opened in your browser automatically (you can disable this by passing --no-browser). The container will be running the code from your local copy of arviz, so you can test your changes.
```

Comment:
If we're being really nitpicky

> local copy of arviz, so 

This should be ArviZ since it's referring to the library

Reply:
You actually don't need to change this one anymore. This section got rewritten in another pr :)